specification/app/resource-manager/Microsoft.App/preview/2022-11-01-preview/examples/ManagedEnvironments_Get.json

```diff
@@ -33,18 +33,25 @@
           "workloadProfiles": [
             {
               "workloadProfileType": "GeneralPurpose",
+              "name": "My-GP-01",
               "minimumCount": 3,
               "maximumCount": 12
             },
             {
               "workloadProfileType": "MemoryOptimized",
+              "name": "My-MO-01",
               "minimumCount": 3,
               "maximumCount": 6
             },
             {
               "workloadProfileType": "ComputeOptimized",
+              "name": "My-CO-01",
               "minimumCount": 3,
               "maximumCount": 6
+            },
+            {
+              "workloadProfileType": "Consumption",
+              "name": "My-consumption-01"
```

Comment:
Fix issues related to `Swagger ModelValidation failed` check. Example needs to have name and like minimum\maximum count for consumption